content/23.vaccines-novel.md

```diff
@@ -62,26 +64,40 @@ Vaccine development programs for COVID-19 emerged very quickly.
-The Coalition for Epidemic Preparedness Innovations (CEPI) is coordinating global health agencies and pharmaceutical companies to develop vaccines against SARS-CoV-2.
+![
+**Structure of the SARS-CoV-2 virus.**
+The development of vaccines depends on the immune system recognizing the virus.
+Here, the structure of SARS-CoV-2 is represented both in the abstract and against a visualization of the virion.
+The abstracted visualization was made using BioRender [@url:https://biorender.com] and the microscopy was conducted by the National Institute of Allergy and Infectious Diseases [@url:https://www.niaid.nih.gov/news-events/novel-coronavirus-sarscov2-images].<!--To Do: Was a template used?-->
+](images/SARS_CoV_2.png){#fig:virus secno=1}
+
+During the development process, one measure used to assess whether a vaccine candidate is likely to provide protection is serum neutralizing activity [@doi:10.1056/nejmoa2022483].<!--Would it make sense to have the definition here?-->
+However, unlike in efforts to develop vaccines for prior viral threats, the duration of the COVID-19 pandemic has made it possible to also test vaccines in phase 3 trials where the effect of the vaccines on a cohort’s likelihood of contracting SARS-CoV-2 was evaluated.
+The Coalition for Epidemic Preparedness Innovations (CEPI) is coordinating global health agencies and pharmaceutical companies to develop vaccines against SARS-CoV-2.<!--This is an important statement, but seems a little lost here near the VE section-->
 
 With vaccine candidates at all stages of development, including full approval of some vaccines, the vaccine development landscape for COVID-19 includes vaccines produced by a wide array of technologies.
 Examining the vaccine development programs tackling the COVID-19 pandemic alongside other 21^st^ century efforts to control emerging viral threats demonstrates the significant biotechnological advances in this field and the importance of modular and adaptable approaches to vaccination.
 In many cases, SARS-CoV-2 is the first pathogen for which these technologies have been approved and administered widely.
-These programs employ a variety of technologies (Figure @fig:vaccines), ranging from established approaches to novel technologies that had never previously gone to market.
+These programs employ a variety of technologies (Figure @fig:vaccines), ranging from established approaches to novel technologies that had never previously gone to market.<!--Note figure not rendering-->
 Development programs using more established technologies are reviewed in a companion manuscript.
 Here, we review the various technologies being explored for the development of SARS-CoV-2 vaccines that use technologies based on nucleic acids.
 
+![
```

Comment:
@RLordan what are your thoughts about dropping this figure? 
https://github.com/greenelab/covid19-review/blob/master/content/images/N002-Vaccines.png
I think it doesn't offer as much now that they are split into separate manuscripts!

Reply:
I think we can drop it considering we are splitting the manuscript. Redundancy is an easy spot for a reviewer